ndb/src/google/cloud/ndb/async_context.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Manage asynchronous context. """
```

Comment:
Have this in `async_context.py` rather than `_eventloop.py` since this is meant to be public facing API.  The implementation bits this relies on are still mostly in `_eventloop.py` since that's where they'll be needed and I'd like to avoid circular imports.